apps/src/sites/studio/pages/layouts/_student_information_interstitial.js

```diff
@@ -0,0 +1,59 @@
+import $ from 'jquery';
+import getScriptData from '@cdo/apps/util/getScriptData';
+import analyticsReporter from '@cdo/apps/lib/util/AnalyticsReporter';
+import {EVENTS} from '@cdo/apps/lib/util/AnalyticsConstants';
+
+const retrieveInfoForCap = getScriptData('retrieveInfoForCap');
+const userId = getScriptData('userId');
+const inSection = getScriptData('inSection');
+
+$(document).ready(function () {
+  const pathName = window.location.pathname;
+
+  if (pathName !== '/lti/v1/authenticate') {
+    $('#student-information-modal').modal('show');
+    retrieveInfoForCap &&
+      analyticsReporter.sendEvent(EVENTS.CAP_STATE_FORM_SHOW, {
+        user_id: userId,
+        in_section: inSection,
+      });
+  }
+
+  const checkInputs = () => {
+    const ageValue = $('#user_age').val();
+    const stateValue = $('#user_us_state').val();
+    $('.age-required').toggle(!ageValue);
+    $('.state-required').toggle(!stateValue);
+    console.log('Age Value: ' + ageValue);
+    console.log('Age Value: ' + stateValue);
+    $('#edit_user #submit-btn').prop(
+      'disabled',
+      ageValue === '' || stateValue === ''
+    );
+  };
```

Comment:
I don't really understand the need for this check:
1. There is no point in removing the asterisk because the field doesn't become optional after selection.
2. Why not simply use HTML5 validation (`required`)?